### DISCUSSION TITLE
Wrong include file hint?

### DISCUSSION THREAD

**Discussion by Klaaktu** [zed-industries/zed#19480] [Feature Requests]
![image](https://github.com/user-attachments/assets/13cf04cd-d216-482a-af54-54b59e7f8f64)
I'm writing C code. Often times when I'm typing something that should be in C headers, I get suggestions for C++ headers instead, and there isn't a suggestion for C header. But then if I try again the issue goes away and C headers are suggested as expected.
In the example, I got suggestions for `<cstdio>` for `FILE` and `fopen()`. On 2nd try `FILE` gave the right hint but `fopen()` was still stuck on `<cstdio>`. After closing and reopening Zed the issue went away.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
